### ISSUE TITLE
Duplicated Titles, is this from a script? Can we use a script to delete?

### ISSUE THREAD

**Issue by davidamichelson** [srophe/britishLibrary-data#1793]
@wlpotter When searching to answer a question for our editors I discovered that a few records have msItems where the English title is repeated.

See for example https://github.com/srophe/britishLibrary-data/blob/7ed9d834c099761265cdcb643468a8d01e06be53/data/tei/46.xml#L152-L156

Any idea on the history of this?

**Comment by wlpotter** [comment]
Yes, likely from a script -- looks to be related to [this change log item](https://github.com/srophe/britishLibrary-data/blob/7ed9d834c099761265cdcb643468a8d01e06be53/data/tei/46.xml#L903).

Probably can use a script to delete though would want to do some testing first to see how many instances

**Comment by davidamichelson** [comment]
I wonder if our editors have found and deleted these?

**Comment by wlpotter** [comment]
In this particular case for 46.xml they did (cf. [this commit](https://github.com/srophe/britishLibrary-data/blob/8bf7f02d3d29980e07e021f8273816a32222cc21/data/tei/46.xml)), not sure in all cases? Would be worth leaving open and checking for duplicated titles

**Comment by wlpotter** [comment]
There are some instances left, but they should be easy to find and remove via script

**Comment by wlpotter** [comment]
I've removed the exact duplicates. I have a sheet of the other cases where we have multiple titles that we or a student can go through to check which are from Wright and which might be uniform titles from our work records.

Here is the spreadsheet we should review: https://docs.google.com/spreadsheets/d/1sIUzfbQwNVxgppDNAzD1Uubb6vwVErL7zCqvG7Cf3-U/edit?gid=0#gid=0
